### ISSUE TITLE
"Convert URL to PDF" button prevents dismiss "cookie policy" on webpages & appears on final PDF conversion blocking content

### ISSUE THREAD

**Issue by HowdyImSWIM** [tawhidmonowar/WebCapture#29]
# **Summary**

The red "Convert URL to PDF" banner/button covers cookie policy pop-ups on websites, preventing their dismissal and causing them to appear in the final PDF output, often blocking content.

## **Description**

**Issue**

- The red "_Convert URL to PDF_" banner/button covers the cookie policy pop-up on websites, preventing dismissal. This **results in the cookie policy being visible in the converted PDF**, often cutting off or blocking content at the bottom of the page.

**TL;DR**

- The red "_Convert URL to PDF_" button **blocks** the dismissal area of cookie policy pop-ups, making them **visible in the PDF** and **cutting off** content behind them.

**Suggested Solution**

- Provide the ability to temporarily hide the red "Convert URL to PDF" button to allow for the dismissal of cookie policies before conversion.

**Comment by tawhidmonowar** [comment]
Thank you, I will try to fix them as soon as possible.
